slash/loader.py

```diff
@@ -187,6 +187,6 @@ def _get_runnable_test_factory(self, thing):
 def _walk(p):
     if os.path.isfile(p):
         return [p]
-    return (os.path.join(dirname, filename)
-            for dirname, _, filenames in os.walk(p)
-            for filename in filenames)
+    return sorted([os.path.join(dirname, filename)
```

Comment:
Also, this doesn't sort the files in a single directory - it sorts *all files* in *all directories*, which is not necessarily desired. I think we should sort each directory separately